### ISSUE TITLE
Inconsistency in AST tructure for block-comments

### ISSUE THREAD

**Issue by romani** [checkstyle/checkstyle#3431]
```
$ cat TableName.java 
/**
* some text
*/
public class TableName {

} 
```

```
$ java -jar checkstyle-7.2-SNAPSHOT-all.jar -J TableName.java 
CLASS_DEF -> CLASS_DEF [4:0]
|--MODIFIERS -> MODIFIERS [4:0]
|   |--JAVADOC -> \n* some text\n<EOF> [1:0]
|   |   |--NEWLINE -> \n [1:0]
|   |   |--LEADING_ASTERISK -> * [2:0]
|   |   |--TEXT ->  some text [2:1]
|   |   |   |--WS ->   [2:1]
|   |   |   |--CHAR -> s [2:2]
|   |   |   |--CHAR -> o [2:3]
|   |   |   |--CHAR -> m [2:4]
|   |   |   |--CHAR -> e [2:5]
|   |   |   |--WS ->   [2:6]
|   |   |   |--CHAR -> t [2:7]
|   |   |   |--CHAR -> e [2:8]
|   |   |   |--CHAR -> x [2:9]
|   |   |   `--CHAR -> t [2:10]
|   |   |--NEWLINE -> \n [2:11]
|   |   `--EOF -> <EOF> [3:0]
|   `--LITERAL_PUBLIC -> public [4:0]
|--LITERAL_CLASS -> class [4:7]
|--IDENT -> TableName [4:13]
`--OBJBLOCK -> OBJBLOCK [4:23]
    |--LCURLY -> { [4:23]
    `--RCURLY -> } [6:0]
```

```
$ java -jar checkstyle-7.2-SNAPSHOT-all.jar -T TableName.java 
CLASS_DEF -> CLASS_DEF [4:0]
|--MODIFIERS -> MODIFIERS [4:0]
|   |--BLOCK_COMMENT_BEGIN -> /* [1:0]
|   |   |--COMMENT_CONTENT -> *\n* some text\n [1:2]
|   |   `--BLOCK_COMMENT_END -> */ [3:0]
|   `--LITERAL_PUBLIC -> public [4:0]
|--LITERAL_CLASS -> class [4:7]
|--IDENT -> TableName [4:13]
`--OBJBLOCK -> OBJBLOCK [4:23]
    |--LCURLY -> { [4:23]
    `--RCURLY -> } [6:0]
```

Attention to:

```
|--MODIFIERS -> MODIFIERS [4:0]
|   |--BLOCK_COMMENT_BEGIN -> /* [1:0]
```

vs

```
|--MODIFIERS -> MODIFIERS [4:0]
|   |--JAVADOC -> \n* some text\n<EOF> [1:0]
```

Expected:

```
    |--BLOCK_COMMENT_BEGIN -> /* [1:0]
    |   |--JAVADOC -> \n* some text\n<EOF> [1:0]
    |   |   |--NEWLINE -> \n [1:0]
    |   |   |--LEADING_ASTERISK -> * [2:0]
    |   |   |--TEXT ->  some text [2:1]
    |   |   |   |--WS ->   [2:1]
    |   |   |   |--CHAR -> s [2:2]
    |   |   |   |--CHAR -> o [2:3]
    |   |   |   |--CHAR -> m [2:4]
    |   |   |   |--CHAR -> e [2:5]
    |   |   |   |--WS ->   [2:6]
    |   |   |   |--CHAR -> t [2:7]
    |   |   |   |--CHAR -> e [2:8]
    |   |   |   |--CHAR -> x [2:9]
    |   |   |   `--CHAR -> t [2:10]
    |   |   |--NEWLINE -> \n [2:11]
    |   |   `--EOF -> <EOF> [3:0]
    |   `--BLOCK_COMMENT_END -> */ [3:0]

```
##

**Comment by romani** [comment]
@kazachka , please update trees at https://github.com/checkstyle/checkstyle/blob/master/src/xdocs/writingjavadocchecks.xml.vm too

**Comment by romani** [comment]
@kazachka , there is still problem in GUI part. 
![screenshot from 2016-10-29 13-56-57](https://cloud.githubusercontent.com/assets/812984/19832691/ab9b5940-9ddf-11e6-8fd3-20e7c9ada3a3.png)

please provide the fix and update http://checkstyle.sourceforge.net/writingjavadocchecks.html#Checkstyle_SDK_GUI with new picture , to build web site you can do "mvn clean site -Dlinkcheck.skip=true".

**Comment by romani** [comment]
issues is closed as UI part will be done at #3445 .
